docs/rules/object-key-newlines.md

```diff
@@ -0,0 +1,68 @@
+# Enforce placing object keys on separate lines (object-key-newlines)
+
+While formatting preferences are very personal, a number of style guides require that
```

Comment:
In this context, "linebreak style" is ambiguous, so not very helpful.

If you think "key" is incorrect, then maybe "property" is the right term to insert instead?